lessons/en/basics/functions.md

```diff
@@ -263,11 +263,11 @@ So now when we call `Greeter2.hello/1`, we can use all of Fred's information:
     iex:15: Greeter2.hello/1
 ```
 
-So we've seen that Elixir pattern-matches at multiple depths because each argument matches against the incoming data independently, leaving us with the variables to call them by inside our function.
+So we've seen that Elixir pattern-matches at multiple depths because each argument matches against the incoming data independently, leaving us with the variables that we can use inside the function.
```

Comment:
Appreciate the suggestion but not sure this is necessarily correct grammar, how about:

```suggestion
So we've seen that Elixir pattern-matches at multiple depths because each argument matches against the incoming data independently, leaving us with variables we can use inside our function.
```